### PR TITLE
Feature/add UI

### PR DESCRIPTION
**Pull request by Estaffieri**
## PR Template 

### Is this a fix or a feature?  
This is a feature and updates our UI to be more intentional.
This also fixes the delete button showing when we intend to remove that functionality.

### What is the change? 
Styling was added for all page views

### What does it fix? 
Removing the delete button from the archive view.

### Where should the reviewer start? 
src/App/App.css Line 1-4, 43-67.
src/Archive/Archive.css Line 1-22.
src/Card/Card.css Line 1-13.
and so on.

### How should this be tested?
Currently we do have a failing test regarding the delete button, but I believe the other branch we are working on fixes that issues.

### CODE REVIEW

File: src/Archive/Archive.js [hunk 1]
```diff
@@ -41,11 +41,13 @@ class Archive extends Component {
     // const key = Date.now()
     return (
       <section>
-        <div>
-          <header>My Saved Advice</header>
+        <header>My Saved Advice</header>
+        <div className="archive-button-container">
+          {/* <header>My Saved Advice</header> */}
```
Comment (ADavidson02) [inline src/Archive/Archive.js:46]: We need to remove this as well in clean up

File: src/Form/Form.css [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    font-family: 'Montserrat', sans-serif;
+    font-size: 1em;
+    text-align: center
+}
+
+form {
+    justify-content: center;
+    align-items: center;
+    margin: 10px;
+
```
Comment (ADavidson02) [inline src/Form/Form.css:12]: We need to remove this space

File: src/Form/Form.js [hunk 3]
```diff
@@ -36,9 +36,9 @@ class Form extends Component {
     return (
       <section>
         <div>
-          <p label="new-thought">New Thought</p>
+          <p className="new-thought">New Thought</p>
```
Comment (ADavidson02) [inline src/Form/Form.js:39]: Do we need to also have a label on this for Aria?

**Review by ADavidson02 — APPROVED** [review]
DO we want to use SASS at all or are we good with using all CSS?

File: src/Card/Card.js [hunk 4]
```diff
@@ -1,16 +1,16 @@
...
-    <section>
+    <section className="advice-card">
       <article key={props.id} className="text-center mb-4">
-        <h3>{props.advice || props.slip.advice}</h3>
+        <h2>{props.advice || props.slip.advice}</h2>
         <p>{props.email}</p>
         <p>{props.name}</p>
       </article>
-      {props.name && <button onClick={() => props.deleteAdviceCard(props)}>Delete</button>}
+      {/* {props.name && <button className="delete-button" onClick={() => props.deleteAdviceCard(props)}>Delete</button>} */}
```
Comment (atownse) [inline src/Card/Card.js:13]: We need to remove this and whatever comments and console.logs are left

**Review by atownse — APPROVED** [review]

**Pull request merged**